labelCloud/io/labels/kitti.py

```diff
@@ -20,22 +23,43 @@ def __init__(
         super().__init__(label_folder, export_precision, relative_rotation)
         self.transformed = transformed
 
+        self.calib_folder = config.getpath("FILE", "calib_folder")
+
     def import_labels(self, pcd_path: Path) -> List[BBox]:
         labels = []
 
         label_path = self.label_folder.joinpath(pcd_path.stem + self.FILE_ENDING)
+        calib_path = self.calib_folder.joinpath(pcd_path.stem + self.FILE_ENDING)
         if label_path.is_file():
+
             with label_path.open("r") as read_file:
                 label_lines = read_file.readlines()
 
             for line in label_lines:
                 line_elements = line.split()
                 centroid = tuple([float(v) for v in line_elements[11:14]])
-                if self.transformed:
-                    centroid = centroid[2], -centroid[0], centroid[1] - 2.3
                 dimensions = tuple([float(v) for v in line_elements[8:11]])
                 if self.transformed:
+
+                    if not calib_path.is_file():
```

Comment:
I added some exception handling here in case the config is missing.

Might improve that to a pop up dialog in the future ...